README.md

```diff
@@ -276,6 +276,22 @@ To clear helm release, do the following:
   }
   ```
 
+#### Clear Helm Release on Failed Build
+
+If your team doesn't need to keep the helm releases for their PR even for failing builds, it is possible to clean them to free further resources.
+This may be useful to reduce resource contention in difficult projects with large teams, where pipeline stability is not yet where it should be.
+
+To clear helm releases for failing builds, do the following:
+
+  ```
+  withPipeline(type, product, component) {
+    ...
+    enableCleanupOfHelmReleaseOnFail()
```

Comment:
It would be good if there was a way so that you don't need to add two lines to cleanup success and failure, i.e. in your screenshot you're having to add two lines of config

You could add something like this as well?
```
enableCleanupOfHelmReleaseAlways()
```

That would map to the existing callback names?
https://github.com/hmcts/cnp-jenkins-library#extending-the-opinionated-pipeline

🚫 At a minimum I would suggest renaming this to match the callbacks:

```suggestion
    enableCleanupOfHelmReleaseOnFailure()
```

Reply:
I'll match the callback names.

As for the additional "always" bit, I would leave further simplifications to your team. The purpose of this PR from my standpoint is simply to finally overcome an issue that is frequently blocking us in civil.